css/index.css

```diff
@@ -1 +1,252 @@
-/* Use your CSS from RWD - I here.  IF you don't have CSS, use the solution CSS from RWD - I*/
\ No newline at end of file
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/*FULL PAGE STYLING*/
+
+.container {
+	margin: 0 auto;
+	max-width: 880px;
+	width: 90%;
+	font-size: 62.5%;
+}
+
+
+/*NAVBAR STYLING*/
+.navbar {
+	display: flex;
+	justify-content: space-between;
+	width: 100%;
+	align-items: flex-end;
+	padding-top: 30px;
+}
+.navbar a {
+	text-decoration: none;
+	color: lightgrey;
+	font-size: 0.8rem;
+}
+.navbar nav {
+	display: flex;
+	justify-content: space-between;
+	width: 100%;
+	margin-right: 8%;
+}
+
+/*CTA SECTION*/
+
+.cta {
+	display: flex;
+	align-items: center;
+	justify-content: space-around;
+	border-bottom: 2px solid black;
+	padding-bottom: 60px;
+	padding-top: 68px;
+}
+
+.cta p {
+	font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+	font-size: 4.8rem;
```

Comment:
The spacing here is a little off. Consider using an extension that automatically cleans it up for you. I believe Sublime does have `Prettify`.